apps/portal/prisma/schema.prisma

```diff
@@ -162,4 +171,136 @@ model ResumesCommentVote {
 // Add Questions project models here, prefix all models with "Questions",
 // use camelCase for field names, and try to name them consistently
 // across all models in this file.
+
+enum QuestionsVote {
+    NO_VOTE
```

Comment:
When will it ever be NO_VOTE? If it's NO_VOTE then there won't be a row for the vote right? Unless you're thinking of setting it to NO_VOTE when a user un-upvote or un-downvote

Reply:
Yeah because I was thinking of just doing an upsert on any changes on votes
cc: @keanecjy

Reply:
I'm not sure if it's a good idea to still keep a NO_VOTE if there's no data meant to be kept. You and @keanecjy can decide and align on one way. I'm personally leaning towards deleting instead of upsert-ing.

Reply:
I think deleting would be better, I don't see any benefit of having the soft delete here since the user has already unvoted, and we don't need to store any unnecessary data. Do let me know if you disagree tho!

Reply:
Make sense! I'll update it as a ```Vote``` enum